src/services/token/writeableTokenHook.ts

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
what's the goal of the MR ? Is this a copy paste of configurable dapp ? what about this comment ?

Reply:
actually this MR is just to confirm the approach of refactoring the token service into hooks. there were three instances which were exposed globally and with respect to that, I have created three hooks.

Reply:
ok but what's the difference with configurable dapp ? what to look for ?

Please add a description somewhere to help the reviewer

Reply:
added description. hooks are created on a similar approach as configurable-dapp. just initializing different classes and exposing respective methods through different hooks.

Reply:
a helpful tip for communicating intents:

1) what are you changing (current state)
2) why are you changing (why is current state unsatisfactory)
3) what change are you introducing (why new state solves 2)
4) tradeoffs of the new approach

Reply:
also example ? it's hard to foresee anything here